common/components/GenerateTransaction.tsx

```diff
@@ -0,0 +1,13 @@
+import { GenerateTransactionFactory } from './GenerateTransactionFactory';
+import React from 'react';
+import translate from 'translations';
+
+export const GenerateTransaction: React.SFC<{}> = () => (
+  <GenerateTransactionFactory
+    withProps={({ disabled, isWeb3Wallet, onClick }) => (
+      <button disabled={disabled} className="btn btn-info btn-block" onClick={onClick}>
+        {isWeb3Wallet ? translate('Send to MetaMask / Mist') : translate('DEP_signtx')}
```

Comment:
I've been meaning to mention, this copy doesn't actually really make sense since clicking this button doesn't do anything with metamask or mist. It's not until the confirmation modal that we call out to web3 to do the transacting. Not necessarily a part of this PR though.